vinyldns/endpoints.go

```diff
@@ -66,8 +66,8 @@ func recordSetEP(c *Client, zoneID, recordSetID string) string {
 	return concatStrs("", recordSetsEP(c, zoneID), "/", recordSetID)
 }
 
-func recordSetChangesEP(c *Client, zoneID string, f ListFilter) string {
-	query := buildQuery(f, "nameFilter")
+func recordSetChangesEP(c *Client, zoneID string, f ListFilterInt) string {
+	query := buildQueryInt(f, "nameFilter")
```

Comment:
We can entirely remove the `nameFilter` parameter as there isn't such filter for recordsetChanges as of now. We can just have:
```
query := buildQueryInt(f)
```